packages/ui/origin-ui-core/src/features/general/actions.ts

```diff
@@ -31,6 +31,7 @@ export interface IEnvironment {
     GOOGLE_MAPS_API_KEY: string;
     MARKET_UTC_OFFSET: number;
     DISABLED_UI_FEATURES: string;
+    SMART_METER_ID: string;
```

Comment:
Where is this used?

Reply:
Will be use for extracting `meterId` from device's `externalDeviceIds` array in the same way we are using `ISSUER_ID` now